src/main/resources/application.yaml

```diff
@@ -44,6 +44,18 @@ spring:
 
 azure:
 darts-gateway:
+  storage:
+    blob:
+      client:
+        connection-string: ${AZURE_STORAGE_CONNECTION_STRING:}
+        ## See https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-tune-upload-download-java
+        block-size-bytes: 10_000_000
+        max-single-upload-size-bytes: 10_000_000
+        max-concurrency: 5
```

Comment:
is this restricting the number of uploads to blob storage at once?
if we have 155 courthouses, would this need to be bigger?
@hemantasharma1129

Reply:
@hemantasharma1129  

max-single-upload-size-bytes is just the maximum size per upload if a file is larger then this value it will be split up into chunks and uploaded in chunks. But if it is less then this then it will be uploaded in one go.
<img width="848" alt="image" src="https://github.com/user-attachments/assets/6f921c73-a238-48d5-86a0-fde80dd7abdb">

I believe max-concurrency relates to the maximum threads the chunked upload can use and is applied per upload not global.
<img width="701" alt="image" src="https://github.com/user-attachments/assets/2a3ed18c-69bf-4b2c-b754-dc5dd9bc1779">
 





These settings were pulled from darts-api :)